### ISSUE TITLE
Typescript version warning

### ISSUE THREAD

**Issue by mitschmidt** [Sergiioo/tslint-defocus#8]
Thanks for this very useful package, I used to commit `fit`s and `fescribe`s on a regular base, but now with tslint-defocus our pre commit hooks and CI pipeline catches these quite well.

However, both with Typescript 2.9.2 and 3.1.1 I am receiving this warning message for npm install:

> tslint-defocus@2.0.5 requires a peer of typescript@^2.x | ^3.x but none is installed. You must install peer dependencies yourself.

Is anybody else having the same issue?

**Comment by owain-llewellyn-sp** [comment]
I'm seeing the same issue:

```
WARN tslint-defocus@2.0.5 requires a peer of typescript@^2.x | ^3.x but none is installed. You must install peer dependencies yourself.
```

Versions

```
"tslint-defocus": "^2.0.5",
"typescript": "~3.1.3",
```

**Comment by akurtovic** [comment]
Seeing the same issue while trying to do update from ng6 to ng7. 

`Package "tslint-defocus" has an incompatible peer dependency to "typescript" (requires "^2.x | ^3.x", would install "3.1.6").`

**Comment by BitTerminator** [comment]
I had this issue too when updating to angular 7.  I had to use the `--force` option to upgrade.  tslint-defocus still works fine. 

https://github.com/angular/angular-cli/issues/10875

**Comment by mitschmidt** [comment]
@Sergiioo maybe it would be advisable to just fix the version reference accordingly?

**Comment by Sergiioo** [comment]
Thanks for the heads up, and I'm sorry it took a while to get round to sorting it out. I've just published version `2.0.6` which fixes this.
